### PR TITLE
fix: handle cue panic when do plan validate

### PR DESCRIPTION
**Pull request by morlay**
to avoid confuse panic https://github.com/dagger/dagger/issues/2326
instead, showing error like

```
5:06PM ERROR system | failed to load plan: actions.test.run._exec.env.TEST1: ( dagger.#Secret & client.commands.echo.stdout & (string | dagger.#Secret) & (string | dagger.#Secret) ): actions.test.run._exec.env.TEST1: incomplete cause disjunction: ( dagger.#Secret & client.commands.echo.stdout & (string | dagger.#Secret) & (string | dagger.#Secret) ): actions.test.run._exec.env.TEST1: incomplete cause disjunction
actions.test.run.env.TEST1: ( dagger.#Secret & client.commands.echo.stdout & (string | dagger.#Secret) ): actions.test.run.env.TEST1: incomplete cause disjunction: ( dagger.#Secret & client.commands.echo.stdout & (string | dagger.#Secret) ): actions.test.run.env.TEST1: incomplete cause disjunction
```

### CODE REVIEW

**Comment by mircubed** [comment]
@sipsma could you review since @aluzzardi is OOO this week?

File: plan/validate.go [hunk 1]
```diff
@@ -52,12 +54,21 @@ func isDockerImage(v *compiler.Value) bool {
 	return plancontext.IsFSValue(v.Lookup("rootfs")) && v.Lookup("config").Kind() == cue.StructKind
 }
 
-func isPlanConcrete(p *compiler.Value, v *compiler.Value) error {
+func isPlanConcrete(p *compiler.Value, v *compiler.Value) (err error) {
```
Comment (TomChv) [inline plan/validate.go:57]: Nit: remove `err`, we are not using that syntax in our codebase.
Is there another way to forward err from the routine to the function's return type?

**Review by TomChv — COMMENTED** [review]
Thanks for your contribution :rocket: Improvement looks good overall, I let other verify that error is correctly formatted.
On my side I just left a question, this do not blocks merge

File: plan/validate.go (hunk 1 above)
Comment (morlay) [reply plan/validate.go:57]: This need for receive panic error.

Comment (morlay) [reply plan/validate.go:57]: In fact. I think we could remove the 
https://github.com/dagger/dagger/pull/2710/files#diff-a0113f2e52d8cf56b126333062d3901a0ba24847d5d9f76d05f74b3dfea21921R133

Comment (TomChv) [reply plan/validate.go:57]: It's okay :D

**Comment by TomChv** [comment]
@aluzzardi @sipsma Waiting for your approval to merge

**Pull request closed without merging**